Tools/LyTestTools/ly_test_tools/cli/find_cpp_library.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Tools for inspecting GitHub CODEOWNERS files
+"""
+
+import os
+
+from ly_test_tools.cli.codeowners_hint import get_codeowners
+
+
+def find_cpp_library_from_test_module_name(folder_to_search, test_module_name):
```

Comment:
I don't believe that all files are added from a cmakelists directly in the directory that contains them, however they *should* be added by one in a parent folder. Likely need to walk up folders in the case of a miss.

Also, this function appears named wrong as it is finding a CMakeLists.txt from a CPP library name.